src/client/localplayer.cpp

```diff
@@ -433,12 +433,14 @@ void LocalPlayer::move(f32 dtime, Environment *env, f32 pos_max_d,
 	}
 
 	/*
-		Check properties of the node on which the player is standing
+		Check properties of the node on which the player is standing or in which the player's feet
 	*/
+	v3s16 pos = getStandingNodePos();
 	const ContentFeatures &f = nodemgr->get(map->getNodeNoEx(m_standing_node));
+	const ContentFeatures &f1 = nodemgr->get(map->getNodeNoEx(pos + v3s16(0, 1, 0)));
```

Comment:
Surely this is more lightweight without line 438 and being:
`nodemgr->get(map->getNodeNoEx(m_standing _node + v3s16(0, 1, 0)));`
?